### DISCUSSION TITLE
Problem of display using subplot

### DISCUSSION THREAD

**Discussion by vincentsassard** [GenericMappingTools/gmt#7735] [Code development discussion]
Hello,

I am trying to display two maps next to each other using subplot. However, instead of displaying alternating black and white stripes, the frame displays one black or white stripe and the rest is only one other color on each axis. This only happens using subplot, using plot works fine. Please find my code underneath the figure that I currently have. Let me know if there's any detail that I can provide to make it more helpful.

![image](https://github.com/GenericMappingTools/gmt/assets/57848677/69feb866-8dfb-48a6-a3cf-d4d84573df57)

```
#!/usr/bin/env bash
#gmt set  HEADER_FONT_SIZE 14p
gmt set GMT_THEME cookbook
#gmt set LABEL_OFFSET 0.2c

# Variables
gmt begin Tectonic_setting_AK
        gmt subplot begin 1x2 -Fs6i -R-165.5/-147/51/61 -JB-156/56/50/60/6i -Bxa4f30 -Bya2f30 -BWNes -Sr+t
                gmt subplot set
                gmt basemap -Bxa4f30 -Bya2f30 #-B+s'Seismicity and Structural Variations Along-Strike'

                gmt subplot set
                gmt basemap -Bxa4f30 -Bya2f30 #-B+s'Seismicity and Structural Variations Along-Strike'
        gmt subplot end
gmt end show
```

**Comment by joa-quim** [comment]
See my answer in https://github.com/GenericMappingTools/gmt/issues/7734
